src/securejoin/bob.rs

```diff
@@ -179,7 +180,7 @@ impl BobState {
             } => {
                 let group_chat_id = match chat::get_chat_id_by_grpid(context, grpid).await? {
                     Some((chat_id, _protected, _blocked)) => {
-                        chat_id.unblock(context).await?;
+                        chat_id.unblock_ex(context, Nosync).await?;
```

Comment:
Blocking groups is implemented by deleting them currently, see `ChatId::block_ex()`. I tested it by hands. Let's avoid extra synchronisation for now. When it will be implemeted otherwise, this should be reviewed completely